### ISSUE TITLE
@pytest.mark.last doesn't work

### ISSUE THREAD

**Issue by sergeychipiga** [ftobia/pytest-ordering#15]
```
20:25:57@5 ~/lira/tmp  $ py.test -v proba.py 
============================================================================= test session starts ==============================================================================
platform darwin -- Python 2.7.9 -- py-1.4.26 -- pytest-2.7.0 -- /Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
rootdir: /Users/lira/tmp, inifile: 
plugins: ordering
collected 4 items 

proba.py::test3 PASSED
proba.py::test1 PASSED
proba.py::test2 PASSED
proba.py::test4 PASSED

=========================================================================== 4 passed in 0.01 seconds ===========================================================================
20:25:59@5 ~/lira/tmp  $ cat proba.py 
import pytest


def test1():
    pass

def test2():
    pass

@pytest.mark.last
def test3():
    pass

def test4():
    pass
```

**Comment by dstd** [comment]
@sergeychipiga looks like [these lines of code](https://github.com/ftobia/pytest-ordering/blob/23a9bd9d143b654ef78f8441e06be1b7abe57378/pytest_ordering/__init__.py#L98-103) should be performed after yielding of `remaining_iter` to conform to declared feature

**Comment by sergeychipiga** [comment]
@dstd you are write unreadable code. I fixed that in #17

**Comment by ftobia** [comment]
Fixed in (I believe) 0.4
